recipes/mauve/meta.yaml

```diff
@@ -16,11 +16,11 @@ source:
 requirements:
   build:
     - ant
-    - conda-forge::openjdk
+    - "conda-forge::openjdk"
   host:
     - mauvealigner
   run:
-    - conda-forge::openjdk
+    - "conda-forge::openjdk"
```

Comment:
If its still installing from `defaults` which is already very wired, I think another dependency has a pinning to the default one or something like that.
By default openjdk should come from conda-forge. Or maybe the conda-forge one is broken and we should fix this one.

Reply:
I've tried quoting 'conda-forge::openjdk' as well but it still comes from defaults. Pinning the version number as fastqc does (issue #8586) solves the problem, but strikes me as being a little fragile.

From the previous issues, the conda-forge jdk is the one that works, and the one in defaults is broken but is tending to get pulled in.

Anyway...I _finally_ have a functioning test, so it's ready to merge if it looks ok. Try not to laugh at the number of commits it took to get to this point...